### PR TITLE
Dosu block

### PR DESCRIPTION
**Pull request by smyzin**
- added text block
- added block with gem
- adaptated to Fold

### CODE REVIEW

File: src/components/GemBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  alignItems('items-center'),
+  justifyContent('justify-start'),
+  margin('mt-20', 'md:mt-0'),
+  width('md:w-96'),
+  height('md:h-96'),
+  overflow('md:overflow-hidden')
+)
+const boldTextWrapper = classnames(
+  alignItems('items-center'),
+  padding('px-1', 'py-0', 'md:px-2', 'md:py-2'),
```
Comment (T-Damer) [inline src/components/GemBlock.tsx:39]: `md:p-2` should be better

File: src/components/GemBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  margin('ml-22.5', 'mt-auto', 'md:mx-auto', 'md:my-auto'),
```
Comment (T-Damer) [inline src/components/GemBlock.tsx:40]: `md:m-auto`

**Review by T-Damer — APPROVED** [review]
lgtm, only one small fix

File: src/components/GemBlock.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/GemBlock.tsx:40]: You're right, thanks

**Pull request merged**